core/src/test/scala/akka/kafka/internal/ProducerTest.scala

```diff
@@ -61,7 +53,7 @@ class ProducerTest(_system: ActorSystem)
 
   def recordAndMetadata(seed: Int) = {
     new ProducerRecord("test", seed.toString, seed.toString) ->
-      new RecordMetadata(new TopicPartition("test", seed), seed.toLong, seed.toLong, Record.NO_TIMESTAMP, -1, -1, -1)
+      new RecordMetadata(new TopicPartition("test", seed), seed.toLong, seed.toLong, new Date().getTime, java.lang.Long.valueOf(-1), -1, -1)
```

Comment:
Actually, I wonder if the date should really be set here, see the upgraded fork by @abesanderson https://github.com/abesanderson/reactive-kafka/commit/f378e50fdaf85b6a6fd7f4ee28988620ab018f7a